scripts/index.js

```diff
@@ -0,0 +1,364 @@
+//Apply Style based on screen Size.
+function loadDeviceStyle(href){
+  let link= document.createElement('link');
+  link.rel ="stylesheet";
+  link.href = href;
+  document.head.appendChild(link);
+}
+
+function applyDeciceStyle (){
+  if (window.innerWidth <= 768) {
+    loadDeviceStyle('./Style/mobile.css');
+} else if (window.innerWidth <= 1024) {
+  loadDeviceStyle('./Style/teblet.css');
+} else {
+  loadDeviceStyle('./Style/desktop.css');
+}
+}
+
+window.addEventListener("resize",applyDeciceStyle);
+window.addEventListener("load",applyDeciceStyle);
+
+//
+
+
+const dataUrl = 'https://raw.githubusercontent.com/hirenkumar91/mYapi/main/recipiAppdata/data.json';
+const priceDataurl ='https://raw.githubusercontent.com/hirenkumar91/mYapi/main/recipiAppdata/priceData.json';
+let recipeObject = [];
+let ingredienTPrice =[];
+
+let addCount = 0;
+
+async function fetchRecipes() {
+  try {
+    const response = await fetch(dataUrl);
+    if (!response.ok) throw new Error(`HTTP error! Status: ${response.status}`);
+    
+    const text = await response.text();
```

Comment:
```suggestion
    const text = await response.json();
```
Then you won't need to use `JSON.parse(text)`

Reply:
done